chain/epoch-manager/src/adapter.rs

```diff
@@ -849,6 +851,11 @@ impl EpochManagerAdapter for EpochManagerHandle {
         if seats > 1 { seats as usize } else { 2 }
     }
 
+    fn genesis_protocol_version(&self) -> ProtocolVersion {
+        let epoch_manager = self.read();
+        epoch_manager.reward_calculator.genesis_protocol_version
```

Comment:
This is probably correct but also seems a bit strange to be going through reward_calculator. Is there any other way to get it?